lms/static/sass/_news.scss

```diff
@@ -19,3 +19,31 @@
     }
   }
 }
+
+.site-status {
```

Comment:
I recommend renaming this to _notifications.scss to better reflect what it now contains.